ci-operator/step-registry/rosa/terraform/e2e/setup/rosa-terraform-e2e-setup-commands.sh

```diff
@@ -26,13 +26,13 @@ EOF
 export TF_VARS
 
 export ARCHIVE_NAME=account-roles-terraform-archive
-export TF_FOLDER=ci/e2e/account_roles_files
+TF_FOLDER=ci/e2e/account_roles_files \
 make apply_folder
 
 # TODO: fix this problem or add here a busy-wait loop that makes sure that the account roles exist
 echo "As a temporary hack, wait a minute for the account roles to get created..."
 sleep 1m
 
 export ARCHIVE_NAME=cluster-terraform-archive
-export TF_FOLDER=ci/e2e/terraform_provider_ocm_files
+export TF_FOLDER="${TF_FOLDER:-ci/e2e/terraform_provider_ocm_files}"
```

Comment:
iiuc this code will apply account-roles twice (because we by default have ``TF_FOLDER='ci/e2e/account_roles_files'`` so you'll never fallback to ``ci/e2e/terraform_provider_ocm_files``)

Reply:
It makes more sense to have two separate steps: ``rosa-terraform-e2e-setup-account-roles`` and ``rosa-terraform-e2e-setup-cluster``

Reply:
fixed

Reply:
"It makes more sense to have two separate steps: rosa-terraform-e2e-setup-account-roles and rosa-terraform-e2e-setup-cluster"  - it was like this in my "bash" code, but u chose o bundle it.  if we change back to what I did, then another step (setting the cluster_name) is required as first step.

Reply:
No connection to your old code 😃 . You can do that similarly with the current one